st2common/st2common/services/workflows.py

```diff
@@ -714,11 +743,22 @@ def refresh_conductor(wf_ex_id):
 @retrying.retry(retry_on_exception=wf_exc.retry_on_exceptions)
 def update_task_flow(task_ex_id, ac_ex_status, ac_ex_result=None, ac_ex_ctx=None, publish=True):
     # Return if action execution status is not in the list of states to process.
-    states_to_process = states.COMPLETED_STATES + [states.PAUSED, states.PENDING]
+    states_to_process = (
+        copy.copy(ac_const.LIVEACTION_COMPLETED_STATES) +
+        [ac_const.LIVEACTION_STATUS_PAUSED, ac_const.LIVEACTION_STATUS_PENDING]
+    )
 
     if ac_ex_status not in states_to_process:
         return
 
+    # Replace/Simplify specific states.
+    states_to_collapse = {
+        ac_const.LIVEACTION_STATUS_DELAYED: ac_const.LIVEACTION_STATUS_DELAYED
```

Comment:
A bit confused on what this line is supposed to do.

Should one of the values be policy delayed, or?

Reply:
Yeah, one of this is suppose to be policy delayed.  Thanks for catching this. I worked on this patch before the scheduler PR was merged.